source/common/config/new_grpc_mux_impl.h

```diff
@@ -56,6 +56,25 @@ class NewGrpcMuxImpl : public GrpcMux,
                             GrpcMuxCallbacks&) override;
   void start() override;
 
+  struct SubscriptionStuff {
```

Comment:
Stuff? :)

Reply:
Not my name, was already there.

Reply:
This was a moderately quick and dirty part of the implementation, meant to be quickly cleaned up (removed actually) by #8478, which turned out to run into way more difficulty than expected. #8478's structuring of GrpcMux could be adapted into here if NewGrpcMuxImpl is going to continue existing as a separate thing for a while. That might be part of @wgallagher's piecewise plan for it.